src/main/java/uk/gov/beis/els/categories/waterheaters/service/WaterHeatersService.java

```diff
@@ -248,6 +276,33 @@ public ProcessedEnergyLabelDocument generateHtml(WaterSolarPackagesForm form, Le
       .asProcessedEnergyLabel(ProductMetadata.WATER_HEATERS_PACKAGE, form);
   }
 
+  public ProcessedEnergyLabelDocument generateFicheHtml(WaterSolarPackagesCalculatorForm form, LegislationCategory legislationCategory){
```

Comment:
probably wont need the `legislationCategory` param

Reply:
Good spot. Will remove